x/x/index.md

```diff
@@ -0,0 +1,5 @@
+# List of artist start with x
```

Comment:
The title should be similar to the one found in the template. Please fix this title.